.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+**Note: for support questions, you are strongly advised to use [Stack Overflow](https://stackoverflow.com/questions/tagged/chisel)**.
+
+This repository's issues are reserved for feature requests and bug reports.
+
+* **Type of issue**
+  - [ ] Bug report
+  - [ ] Feature request
+  - [ ] Other enhancement
+
+* **If the current behavior is a bug, please provide the steps to reproduce the problem:**
+  * **What is the current behavior?**
+  * **What is the expected behavior?**
+  * **Please tell us about your environment:**
+    (examples)
+    - version: `3.0-SNAPSHOT`
+    - OS: `Linux knight 4.4.0-92-generic #115-Ubuntu SMP Thu Aug 10 09:04:33 UTC 2017 x86_64 x86_64 x86_64 GNU/Linux`
+
+* **What is the use case for changing the behavior?**
+
```

Comment:
I think **DevelopmentPhase** should still be in here, with "request" and "proposal" as possible values.